packages/scripts/src/lib/plugin/handlers/fields/calculations.ts

```diff
@@ -48,33 +50,31 @@ class Calculation implements FreeformHandler {
       const doCalculations = () => {
         const allVariablesHaveValues = Object.values(variables).every((value) => value !== '');
 
-        if (!allVariablesHaveValues) {
-          return;
-        }
+        if (!allVariablesHaveValues) return;
```

Comment:
Lets stick to our code style and refrain from using one line conditional rules.

```suggestion
        if (!allVariablesHaveValues) {
          return;
        }
```